### DISCUSSION TITLE
Sort comments by date; ascending / descending

### DISCUSSION THREAD

**Discussion by pnkellogg** [TeamNewPipe/NewPipe#10830] [Questions]
First off, abundant kudos for this front-end; responsive, scalable, robust. _Thanks much_, indeed.

At times, it may be conducive to have YouTube comments sorted by date. Maybe some eager soul has got the capacity to implement this?

**Comment by AudricV** [comment]
> responsive, scalable, robust

I have a lot of doubts about that, otherwise #10118 wouldn't have exist :P

Your discussion is a duplicate of #3115 and #9423. Please search (better, with different keywords?) for existing discussions and issues the next time you want to open a new discussions. Thank you.
